app/assets/stylesheets/thredded/components/_mention-autocomplete.scss

```diff
@@ -0,0 +1,31 @@
+ul.textcomplete-dropdown {
```

Comment:
We should namespace the class names for our styles to `thredded-` (like we do with select2) in case the user is also using jquery.textcomplete in main_app.